specification/search/resource-manager/Microsoft.Search/stable/2023-11-01/search.json

```diff
@@ -2341,6 +2437,61 @@
         "type"
       ],
       "description": "Identity for the resource."
+    },
+    "QuotaUsagesListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/QuotaUsageResult"
+          },
+          "description": "The quota usages for the SKUs supported by Azure Cognitive Search."
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Request URL that can be used to query next page of quota usages. Returned when the total number of requested quota usages exceed maximum page size."
+        }
+      },
+      "description": "Response containing the quota usage information for all the supported skus of Azure Cognitive Search service.",
+      "x-ms-external": true
+    },
+    "QuotaUsageResult": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The resource id of the quota usage sku endpoint for Microsoft.Search provider."
+        },
+        "unit": {
+          "type": "string",
+          "description": "The unit of measurement for the search sku."
+        },
+        "currentValue": {
+          "type": "integer",
+          "description": "The currently used up value for the particular search sku."
+        },
+        "limit": {
+          "type": "integer",
+          "description": "The quota limit for the particular search sku."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "object",
+          "properties": {
+            "value": {
+              "type": "string",
```

Comment:
Can this be ""$ref": "#/definitions/Sku"?

Reply:
No, the attributes here are specified by ARM which is different from definitions/Sku.